webextensions/api/proxy.json

```diff
@@ -498,7 +498,8 @@
               ],
               "firefox_android": [
                 {
-                  "version_added": "56"
+                  "version_added": "56",
+                  "version_removed": "71"
```

Comment:
Version removed markup was put in for FF desktop in version 71 back in #4496. The associated issues doesn't explicitly talk about the removal in android, but it seems very unreasonable to assume that it was not removed. So I think it was an omission. Added here to register, and below in unregister.

Reply:
Changed to 79 (first valid release)